rest/endpoints/license/management_license.yml

```diff
@@ -0,0 +1,88 @@
+openapi: 3.0.0
+info:
+  description: >
+    This endpoint provides licenses details.
+    Basically, it will show all licenses with pagination. Besides, it can be filtered to limit results.
+  version: 1.0.0
+  title: Management
+tags:
+  - name: Licenses
+    description: |
+      Licenses Management API is used for management level, not for clients.
```

Comment:
Is it supposed to some IP whitelist mechanism for management APIs? I cannot find it from the `LicenseController`.